set.mm

```diff
@@ -341639,6 +341684,51 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
       KJZWBWAGVGRRGGVBVCVDVESZVHDEVLVNWCVIVJVK $.
   $}
 
+  ${
+    $d A x y z $.  $d F x y $.  $d G x y $.  $d X x y z $.  $d ph x y z $.
+    efabl.1 $e |- F = ( x e. X |-> ( exp ` ( A x. x ) ) ) $.
+    efabl.2 $e |- G = ( ( mulGrp ` CCfld ) |`s ran F ) $.
+    efabl.3 $e |- ( ph -> A e. CC ) $.
+    efabl.4 $e |- ( ph -> X e. ( SubGrp ` CCfld ) ) $.
+    $( The image of a subgroup of the group ` + ` , under the exponential
+       function of a scaled complex number, is an Abelian group.  (Contributed
+       by Paul Chapman, 25-Apr-2008.)  (Revised by Mario Carneiro,
+       12-May-2014.)  (Revised by Thierry Arnoux, 26-Jan-2020.) $)
+    efabl $p |- ( ph -> G e. Abel ) $=
```

Comment:
This theorem is jumping the gun a bit. Before concluding that G is an abelian group, you want to know that `ran F` is a subgroup of the unit group of CCfld (or a submonoid of multiplicative CCfld).

Reply:
Can do! I've added that proof.